### PR TITLE
Update read me and makefile

### PR DESCRIPTION
**Pull request by jeremyrhyde**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export PATH := ${GOPATH}:$(PATH) 
+SHELL := /usr/bin/env bash 
+
+default: setup build
+
+setup: install-dependencies
+
+install-dependencies:
+ifneq (, $(shell which brew))
+	brew update
```
Comment (thegreatco) [inline Makefile:17]: Brew is not needed to build this module, why are we running brew update?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+else ifneq (, $(shell which apt-get))
+	sudo apt-get update
```
Comment (thegreatco) [inline Makefile:19]: This command isn't doing anything useful

**Review by thegreatco — CHANGES_REQUESTED** [review]
I'm not sure why we have a makefile at all for this. Building with the native go commands is simple enough. 

I also think we should explain all the fields in the readme if we're adding one.

File: Makefile (hunk 1 above)
Comment (jeremyrhyde) [reply Makefile:17]: The Makefile I included is just a framework I was using in eng when creating modules. It helped making things consistent as I jumped from module to module, however, you are right that it is not necessary in this case. Would you like me to remove it from this PR?

**Comment by jeremyrhyde** [comment]
Closing PR as readme has been added and Makefile deemed unnecessary

**Pull request closed without merging**